tox.ini

```diff
@@ -17,7 +16,9 @@ sitepackages=True
 deps =
     pytest
     pytest-cov
-    pytest-django
```

Comment:
pytest-django modifies the PYTHONPATH in https://github.com/pytest-dev/pytest-django/blob/a67ab3bd0eda8a2b76b4c5954b04482b4f27e20a/pytest_django/plugin.py#L123

This means when we switch over to Flask, the tox.ini needs the PYTHONPATH explicitly setup to point at /app